### PR TITLE
Update PureScript to v0.11.7

### PR DESCRIPTION
**Pull request by TimothyKlim**
Release notes: https://github.com/purescript/purescript/releases/tag/v0.11.7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "purescript",
-  "version": "0.11.6",
+  "version": "0.11.7",
```
Comment (shinnn) [inline package.json]: https://twitter.com/sindresorhus/status/298376330544824320

Comment (TimothyKlim) [reply package.json]: fixed

**Comment by shinnn** [comment]
Please add a link to the v0.11.7 release note in the same way as other version bump PRs. https://github.com/purescript-contrib/node-purescript-bin/pulls?q=Bump

**Comment by TimothyKlim** [comment]
@shinnn added, thanks!

**Comment by shinnn** [comment]
Pending by https://github.com/mafintosh/pump/pull/23.

**Comment by shinnn** [comment]
Published to [npm](https://www.npmjs.com/package/purescript).

```console
$ npm view purescript version
0.11.7
```

**Pull request merged**